php/php.refactoring/src/org/netbeans/modules/refactoring/php/findusages/PhpWhereUsedQueryPlugin.java

```diff
@@ -63,6 +64,10 @@ public Problem prepare(final RefactoringElementsBag elementsBag) {
                 if (fileObject == null) {
                     continue;
                 }
+                if (cancelled) {
```

Comment:
@tmysik Instead, should use `AtomicBoolean.compareAndSet()`?

Reply:
We should leave a comment about why we change it to `false`.

Reply:
Added comment why `cancelled` is reset.

For `cancelled` I used same type as in OccurencesFinderImpl in PHP Editor:
https://github.com/apache/netbeans/blob/514e170f43416aec1a25c1c443837bf1ef9043f0/php/php.editor/src/org/netbeans/modules/php/editor/csl/OccurrencesFinderImpl.java#L62

Reply:
Volatile is OK here, IMHO.